tests/fixtures/marketplace-guest-production-baseline.sql

```
@@ -0,0 +1,127 @@
+  reference_id text not null unique,
+  created_at timestamptz not null default now(),
+  updated_at timestamptz not null default now(),
+  constraint quote_requests_status_check check (
+    status in (
+      'draft', 'submitted', 'pending_review', 'approved', 'matched', 'answered',
+      'booked', 'completed', 'cancelled', 'rejected'
+    )
+  )
+);
+
+create index quote_requests_status_idx on quote_requests (status);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)